aws/data_source_aws_route53_resolver_rule.go

```diff
@@ -0,0 +1,152 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
Lets fix the ordering of the github imports. I believe if you remove the blank line between the github imports gofmt should fix the ordering automatically.

Reply:
Done.